feathr_project/test/test_feature_registry.py

```diff
@@ -69,7 +69,7 @@ def test_feathr_register_features_partially(self):
         client: FeathrClient = registry_test_setup(os.path.join(test_workspace_dir, "feathr_config.yaml"))
         client.register_features()
         time.sleep(30)
-        full_registration = client.get_features_from_registry(client.project_name)
+        [full_registration, keys] = client.get_features_from_registry(client.project_name, return_keys = True, verbose = True)
```

Comment:
should this be a list? Might better to be a tuple?

Reply:
> should this be a list? Might better to be a tuple?

For what reason?

Reply:
> > should this be a list? Might better to be a tuple?
> 
> For what reason?

Tuples are not immutable and I think that's what we meant here - basically return two dicts, the first dict indicate it's for keys and the second dict is for features. List is mutable so it might be a bit confusing here.

Reply:
Changed